fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1466,11 +1473,14 @@ ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 						    encryptedMutation.present()) {
 							backupMutation = encryptedMutation.get();
 						} else {
-							std::pair<EncryptCipherDomainName, EncryptCipherDomainId> p =
-							    getEncryptDetailsFromMutationRef(self->pProxyCommitData, backupMutation);
-							EncryptCipherDomainId domainId = p.second;
-							backupMutation =
-							    backupMutation.encrypt(self->cipherKeys, domainId, arena, BlobCipherMetrics::BACKUP);
+							// Avoid enumerating mutations if cluster only serves default encryption domains
+							if (pProxyCommitData->tenantMap.size() > 0) {
```

Comment:
Wait doesn't this make it so that these clear range backup mutations are not encrypted when the cluster serves default encryption domains?

Reply:
Thanks for pointing out, the code snippet seems incorrect, the optimization is meant for `getResolution` to avoid enumerating mutations as part of handling commit- transactions. 

Will update the code to do the desired thing.